cibuild.sh

```diff
@@ -234,7 +234,8 @@ function rx-gcc-toolchain {
   if [ ! -f "$prebuilt/renesas-toolchain/rx-elf-gcc/bin/rx-elf-gcc" ]; then
     case $os in
       Linux)        
```

Comment:
Should we add Darwin and merge renesas into avr-mips-riscv-x86-xtensa?

Reply:
That will require building gcc on mac every build which we probably don't want same with windows. I thinking we probably need a pipeline for building these tool chains and uploading them to S3 or something. I want that functionally for doing the bloaty stuff anyway.  Are you fine just kicking this down the road a bit? It should still finish before the Mac worker anyway.

Reply:
With Windows Cygwin I think most people use https://developer.arm.com/tools-and-software/open-source-software/developer-tools/gnu-toolchain/gnu-rm

There are lots of other Windows native toolchains.

Reply:
But that does not have the rx support and Renesas like Microchip have the binary releases behind a login which is really annoying.

Reply:
I have been think about to add option into datalist to filter out some build from the specific OS, something like this:
```
/renesas/rx65n/rx65n-grrose, linux
```
so we can get the fine-grained control.

Reply:
Could be nice. That way we could also enable some of the sim builds that are blocked because they don't support macOS.

In the meantime can we just leave this as the additional target and then reshuffle them once we have the feature?

Reply:
Sure, let's go ahead merge this.